skip-tests.json

```diff
@@ -644,6 +644,7 @@
     },
     "coinex": {
         "skipWs": "timeouts",
+        "skipCSharp": "service busy: https://app.travis-ci.com/github/ccxt/ccxt/builds/269539760#L3513",
```

Comment:
why only c# then?

Reply:
i see such scenarios more frequent, when the exchange connection timeouts/fails in many times only in c#, but not in others. so I just suppose there might be some underlying lib/header/etc for c#, as it's relatively newer and there might be somethings in the backgrounds. however, as a quick workaround, we can only skip in c# for now and observe what happens.

Reply:
here another example: https://app.travis-ci.com/github/ccxt/ccxt/builds/269575006#L3639